Src/library/base/ise/kernel/dotnet/file.e

```diff
@@ -165,6 +165,22 @@ feature -- Initialization
 			open_append: is_open_append
 		end
 
+	make_temporary_open
```

Comment:
what about  make_temporary_open_with_prefix ?
is it available for dotnet?

Reply:
Not sure need to check.